### ISSUE TITLE
Creating credentials with amplify configure doesn't properly add spaces.

### ISSUE THREAD

**Issue by returnzer0** [aws-amplify/amplify-cli#686]
**Describe the bug**
When I create a new profile with `amplify configure`, it adds the credentials in  ~/.aws/credentials but fails to add spaces on either side of the `=` like below.

```
aws_access_key_id=XXXXXXXXX
aws_secret_access_key=XXXXXXXXXX
``` 

**To Reproduce**
Steps to reproduce the behaviour:
1. Run `$ amplify configure`
2. Go through setup and add a new user in the console
3. Enter `access_key_id` and `aws_secret_access_key` when prompted
4. Run `$ cat ~/.aws/credentials` and you will see no spaces between the newly created profile creds.

**Expected behavior**
In order for `amplify init` to run properly and create resources, there should be spaces added on both sides of the `=` sign for the AWS Profile. In case there is no spacing it causes the error below:

```
{ Error: connect ETIMEDOUT 169.254.169.254:80
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1117:14)
  message: 'Missing credentials in config',
  errno: 'ETIMEDOUT',
  code: 'CredentialsError',
  syscall: 'connect',
  address: '169.254.169.254',
  port: 80,
  time: 2019-01-10T00:27:09.499Z,
  originalError:
   { message: 'Could not load credentials from any providers',
     errno: 'ETIMEDOUT',
     code: 'CredentialsError',
     syscall: 'connect',
     address: '169.254.169.254',
     port: 80,
     time: 2019-01-10T00:27:09.499Z,
     originalError:
      { errno: 'ETIMEDOUT',
        code: 'ETIMEDOUT',
        syscall: 'connect',
        address: '169.254.169.254',
        port: 80,
        message: 'connect ETIMEDOUT 169.254.169.254:80' } } }
```

**Desktop:**
 - OS: Mac OS Mojave 10.14.2 (18C54)
 - Version 0.1.40

**Comment by UnleashedMind** [comment]
space around = sign is not needed, see for example this aws cli [doc](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
What command did you run to get that error? 
Please get the latest version and try again, I published a bug fix yesterday and the latest version is now 0.1.41

**Comment by returnzer0** [comment]
I ran `$ amplify init` (to initialize a new project) after running `$ amplify configure` and that is what caused the error above. 

I ended up deleting the `~/.aws/credentials` and `~/.aws/configure` then used the aws cli by using`$aws configure` to set up the profile again, instead of using `$ amplify configure`, and this time it worked and there were also no spaces present.

**Comment by UnleashedMind** [comment]
@returnzer0 
so you went through these steps to get the error? 
- `amplify configure` to setup a profile
- `amplify init` and select the newly created profile to init a project

I was not able to reproduce the error though. Please let us know the specifics of what command printed out that error, if you want us to investigate further.

**Comment by returnzer0** [comment]
Yes, sorry if the previous comment wasn't clear.

Here is the chain of commands that I performed:

- `$ npm install -g @aws-amplify/cli` to install the cli
- `$ amplify configure` to steup a profile
- `$ amplify init` to start the project, went through the setup and encoutered error above at the very end when it tried to build the resources

I just tried the commands again with version 0.1.41 and it works perfectly fine now! So, will close this issue.
